ovs/datapath.go

```diff
@@ -0,0 +1,338 @@
+// Copyright 2021 DigitalOcean.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ovs
+
+import (
+	"errors"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var (
+	errMissingDataPathName  = errors.New("datapath name argument is mandatory")
+	errUninitializedClient  = errors.New("client unitialized")
+	errMissingMandatoryZone = errors.New("at least 1 zone is mandatory")
```

Comment:
you use "errMissingDataPathName" for a mendatory name, but "errMissingMendatoryZone" for the mendatory zone. To align both can we use "errMissingMendatoryZone" or "errMissingMendatoryDataPathName" ?